.travis.yml

```diff
@@ -21,21 +21,19 @@ addons:
       - docker-ce
 
 env:
+  global:
+    - secure: "RKBtSjs0cgUYZXbowIuE2iJKTYrNbM6FbBZLCc3VXpbMoaDArtlHVuB3QWyzrbXFBVZhASXlST26VTb/1I5+FrMY+OLV4ei2kHIxM/rG0q8ymycpuxcV7I8cq2p952tSMwdPgVkW9Dy4N0k/me88cNRPIB71LJLmsRYlqVvXxmH9i17hRT50LzzfJDw0QpwN9Z+mvI3Pp/IgInlbBEnm2iMluYvbC6akuoP+WBh8pAMCGMm5/oZ0tCMvd+L/msXjM1BFApUaO+/bTRqDjsZeKy+VKoZ4AHc7A2oHqZrI6NfmnUxA0UwZcfDMXTzCwmaVZwzjjEu6Ykq4TnC8U4oeiz4B2802kFfwyHSSUcMKPnZKONzkCCqi1nZWsdJPOy5lK+QpMd1t+kZiHe4UdutVMb8H4Yaqiy0gqU90vujvgxp9ly++FRYIbR5NJPVw6MZaNmm6DYgjyqSn4Ke93McMFHFpkIjpsHFx5rOFdrZXDzg8aDQKKE4YdNWH4y+qGEKgwKkNxl/0pa5r0wBRmrxeSDZMPHpRnBL62Ch4YB30chWz5WWYEM+n0gl2ExGSpbUePAzjQEesl9kcbEdTnN1N4zOviW4PqJVrLP7WAvOKO1Xs83td7SeVniDTRwPUBjEUzPLfXxS0oh6hzFshYhV0FQTVu+xGSFx1pdrn+RUAGRs="
+    - secure: "N81+mfPpeor7AxHbuoRBZi6eqNoWTGnb+u5TtV4PxxWG5/+ipbCsUvb2wUk3ZYdfeWh611DDksTZ/HrUTpXG/pccZZ/Pl6A4CPf050yZ7OXa1gF9x1rhMRd5hwadmJTiGBKqEMxpojO45PhPz/IzUurItOM6VUGHKEHSieyoP1ml+8a0z0f/MTcV8UVYBSo7cWgXfmWOcBrAv9GevbN/t2UEbXqccutVG6GpcVf/ZDwPo1ezNtyiDq8ChVOR6liO0m8SlygMUMUGwlJ0OvtYfanJ2WcH0z3bQpGnkTnawz3/Cz+W/55MkJ19iYYLPFjN6tqxDCtC9KmtrRzixueZSS9b5NdRg6vo3ry2XuBamYHgLKpxZoEUrD3Xs3NaFQwxn19cFo8O/FmEgX4HmNxRgiwemCN9CYOGdwHCFX3B/xnXo6qX6iTK6++P+w/tl3mEQHrutEjXZSLrjh56hYk/1FovcS3mx5WSMjoEOK3pdrbd1Mp0rjXZPIo0zKfz3Z2xtBg7vTmMp/Bryd/REns/hrZDynTD/e7GloXpNe477PM04lbzHc2tv4KnGiw9hwJsuKw8jx067bMOrKV1Xrb5cKK8uAfaO8CXL8zDNNlv4WLbptnuxf32UBa6FcP6rB6/mnj2ndg6oVMPbMAA3av7bKgM5JJ8TfHAKNnPGFzsbw8="
```

Comment:
ok, so this stuff is just like a secure thing having to do with aws credentials that can be stored publicly, correct?

Reply:
Yeah, it's travis' encryption of variables. No matter what I give them it's that long. The first one is the public and the second is the secret key